Documentation/ApiOverview/Events/Events/Core/Imaging/_ModifyRecordOverlayIconIdentifierEvent/_ModifyRecordOverlayIconIdentifierEventListener.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Vendor\MyPackage\Core\EventListener;
```

Comment:
```suggestion
namespace Vendor\MyPackage\Imaging\EventListener;
```

This is a better namespace than "Core"